.gitignore

```diff
@@ -23,6 +23,4 @@ test/htmlcov
 
 # Docker
 dockerfiles/id_rsa*
-dockerfiles/authorized_keys
-
-venv/
+dockerfiles/authorized_keys
```

Comment:
This change can be reverted, I believe? Unless there is a reason for not ignoring `venv/`.